packages/signature-controller/package.json

```diff
@@ -71,7 +71,7 @@
   },
   "peerDependencies": {
     "@metamask/approval-controller": "^7.0.0",
-    "@metamask/keyring-controller": "^17.0.0",
+    "@metamask/keyring-controller": "^18.0.0",
```

Comment:
Similar, can we include this in the release?

Reply:
Major or minor?

Reply:
Major.